Packs/FlashpointFeed/Integrations/IgniteFeed/IgniteFeed.py

```diff
@@ -0,0 +1,600 @@
+""" Flashpoint Ignite Feed Integration for Cortex XSOAR (aka Demisto) """
+
+""" IMPORTS """
+
+import traceback  # noqa E402
+from typing import Any, Dict, Tuple  # noqa E402
+
+import urllib3  # noqa E402
+
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+# Disable insecure warnings
+urllib3.disable_warnings()  # pylint: disable=no-member
+
+''' CONSTANTS '''
+
+DATE_FORMAT = "%Y-%m-%dT%H:%M:%SZ"  # ISO8601 format with UTC, default in XSOAR
+LIMIT = 10
+MAX_INDICATORS = 10000
+MAX_FETCH = 1000
+DEFAULT_FIRST_FETCH = '3 days'
+CURRENT_TIME = 'now'
+DEFAULT_OFFSET = 0
+DEFAULT_SORT_ORDER = 'asc'
+DEFAULT_INDICATOR_TYPE = 'Flashpoint Indicator'
+FILE_TYPES = ['sha1', 'sha256', 'sha512', 'md5', 'ssdeep']
+TIMEOUT = 60
+STATUS_LIST_TO_RETRY = (429, slice(500, 600))
```

Comment:
Can you explain how this is working, Are you expected to get the all status codes between 500 to 600? As i understand the slice will not return such a list you can use instead the range function as mentioned [here](https://stackoverflow.com/questions/51700887/python-requests-retry-for-any-status-code-but-not-400)

<img width="378" alt="Screenshot 2024-07-05 at 0 02 04" src="https://github.com/demisto/content/assets/78307768/c210f57f-68e1-4d8e-9fe2-ca85841ff987">

Reply:
We're trying to use the whole range, however proposed solution for status used from requests looks more efficient.